android/src/main/java/io/sentry/capacitor/SentryCapacitor.java

```diff
@@ -12,25 +12,33 @@
 import io.sentry.Breadcrumb;
 import io.sentry.HubAdapter;
 import io.sentry.Integration;
+import io.sentry.IScope;
 import io.sentry.Sentry;
 import io.sentry.SentryEvent;
 import io.sentry.SentryLevel;
+import io.sentry.SentryOptions;
 import io.sentry.UncaughtExceptionHandlerIntegration;
 import io.sentry.android.core.BuildConfig;
 import io.sentry.android.core.AnrIntegration;
+import io.sentry.android.core.InternalSentrySdk;
+import io.sentry.android.core.SentryAndroidOptions;
 import io.sentry.android.core.NdkIntegration;
 import io.sentry.android.core.SentryAndroid;
 import io.sentry.protocol.SdkVersion;
 import io.sentry.protocol.SentryPackage;
 import io.sentry.protocol.User;
-import java.io.File;
-import java.io.FileOutputStream;
+
 import java.io.UnsupportedEncodingException;
+import java.util.Date;
 import java.util.HashMap;
 import java.util.Iterator;
 import java.util.List;
+<<<<<<< HEAD
+import java.util.Map;
+=======
 import java.util.Set;
 import java.util.UUID;
+>>>>>>> cf9f7218245fecae3a21f2e6612b0931af53939a
```

Comment:
Unresolved merge conflict

```suggestion
import java.util.Map;
import java.util.Set;
import java.util.UUID;
```